user_modules/shared_testcases/testCasesForPolicyTable.lua

```diff
@@ -902,9 +902,8 @@ function testCasesForPolicyTable:flow_SUCCEESS_EXTERNAL_PROPRIETARY(self, app_id
   if (ptu_file_name == nil) then ptu_file_name = "PolicyTableUpdate" end
   if (ptu_file == nil) then ptu_file = "ptu.json" end
   --[[Start get data from PTS]]
-  --TODO(istoimenova): function for reading INI file should be implemented
-  --local SystemFilesPath = commonSteps:get_data_from_SDL_ini("SystemFilesPath")
-  local SystemFilesPath = "/tmp/fs/mp/images/ivsu_cache/"
+  local SystemFilesPath = commonFunctions:read_parameter_from_smart_device_link_ini("SystemFilesPath") .. "/"
```

Comment:
@dboltovskyi, may be will be better if you use commonFunctions:pathJoin at the specific place of call, not to add here.